groups/nts/ntsa/ntsa_data.cpp

```diff
@@ -1867,6 +1869,62 @@ struct DataUtilImpl {
     /// Pop the specified 'numBytes' from the specified 'data' assuming
     /// 'data' is represented as a file.
     static void popFile(ntsa::Data* data, bsl::size_t numBytes);
+
+    /// *** Copy ***
+
+    /// Copy of the specified 'source' into the specified destination. Return
```

Comment:
Missed single quotes around `destination`. Here and below in the file

Reply:
Done.